config/boards/sipeed-licheepi-4a.conf

```diff
@@ -0,0 +1,63 @@
+# RISC-V LicheePi 4A
+BOARD_MAINTAINER="chainsx kreyren"
+
+# Metadata
+BOARD_NAME="LicheePi 4A"
+BOARDFAMILY="thead"
+
+# OpenSBI
+case "$BRANCH" in
+	"legacy")
+		# RevyOS is used as the official distro for the board and this repository is where upstream affiliates work on mainline to original project
+		OPENSBISOURCE="https://github.com/revyos/thead-opensbi"
+		OPENSBIBRANCH="branch:lpi4a"
+		OPENSBI_USE_GCC="> 6.3"
+		OPENSBI_TARGET_MAP=";;build/platform/generic/firmware/fw_dynamic.bin"
+		;;
+	*)
+		display_alert "Building the supervisor binary for the board '$BOARD' is not implemented for branch '$BRANCH', success is not guaranteed" warn
+esac
+
+# Bootloader
+case "$BRANCH" in
+	"legacy")
+		BOOTSOURCE='https://github.com/chainsx/thead-u-boot'
+		BOOTBRANCH='branch:extlinux'
+		BOOTPATCHDIR="legacy"
+		BOOT_FDT_FILE="thead/light-lpi4a.dtb"
+		BOOTCONFIG="light_lpi4a_defconfig"
+		BOOTFS_TYPE="ext4"
+		UBOOT_TARGET_MAP=";;u-boot-with-spl.bin"
+		;;
+	*)
+		display_alert "Building bootloader for the board '$BOARD' is not implemented for branch '$BRANCH', success is not guaranteed" warn
+esac
+
+# Kernel
+KERNEL_TARGET="legacy"
+
```

Comment:
I try to work on current and edge kernel implementation in different merge request